common/main_test.cpp

```diff
@@ -0,0 +1,21 @@
+#include <gtest/gtest.h>
+
+#include <common/log.hpp>
+
+using namespace testing;
+
+namespace log = mender::common::log;
+
+class GlobalTestEnvironment : public ::testing::Environment {
+public:
+	void SetUp() override {
+		log::SetLevel(log::LogLevel::Trace);
```

Comment:
While you're at it, could you also move [this call](https://github.com/mendersoftware/mender/blob/feature-c%2B%2B-client/common/events_io_test.cpp#L39) to here, and remove it from that specific test?